.nvmrc

```diff
@@ -0,0 +1 @@
+v12.22.8
```

Comment:
Did we enforce this version somewhere?

Reply:
Currently, neither the documentation nor the code have any reference guiding/indicating which version of Node.js to use when working with the agent. Nevertheless, since this [PR](https://github.com/elastic/apm-agent-rum-js/pull/935) we are establishing version 12 for several steps of the pipeline. 

I extracted that version added there from logs of Jenkins. I added the exact version to have sync between development and pipeline. But maybe I'm being too picky, at the end of the day adding "12" might be enough since a tool such as nvm will install the latest version of 12 anyway... (there is a [12.22.9 version available](https://user-images.githubusercontent.com/15065076/148920046-9b70d28d-ebf6-4393-bf20-0c6d7e602ca9.png) since January 1st)

What do you think?

Reply:
Gotcha, Yeah just 12 works for me.